querybook/webapp/components/DataDocStatementExecution/StatementResult.scss

```diff
@@ -61,4 +61,19 @@
 .StatementResult-column-toggle-menu {
     max-height: 200px;
     overflow-y: auto;
+    .hide-column-search-bar-wrapper {
+        .SearchBar {
```

Comment:
i see the same css repeated in https://github.com/pinterest/querybook/pull/1136#pullrequestreview-1276202682
can you make it a shared class under SearchBar.scss?

Reply:
Should all SearchBars be formatted in this way? I see that some of these properties in the current SearchBar.scss have different property values already like flex.